Homework8/Homework8/Truck.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework8
+{//3
+    public class Truck : Vehicle, IDriveable
```

Comment:
Jen fyi -> uz trida `Vehicle` muze implementovat interface `IDriveable`. Tady by se ti to pak zjednodussilo a mel bys tu jen
`Truck : Vehicle` :) 

Tenhle komentar plati pro vsechny tridy